sonar-bungee/src/main/java/xyz/jonesdev/sonar/bungee/fallback/FallbackListener.java

```diff
@@ -42,6 +43,9 @@ public void handle(final @NotNull LoginEvent event) {
     // the connecting player is greater than the configured amount
     final int maxOnlinePerIp = Sonar.get().getConfig().getMaxOnlinePerIp();
 
+    // Not FallbackInitialHandler. It just original InitialHandler.
+    final InitialHandler original = (InitialHandler) event.getConnection();
+    final FallbackInitialHandler initialHandler = Objects.requireNonNull(FallbackInitialHandler.recordMaps.get(original));
```

Comment:
I'm looking for a way to replace it. or make the required method a static method.